rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": "#acceptAllMain" }],
+  "optOut": [
+    { "click": ".sibbo-panel__aside__buttons a[data-nav=\"purposes\"]" },
+    { "click": ".sibbo-panel__main__header__actions a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": ".sibbo-panel__main__footer__actions a[data-focusable=\"next\"]" },
+    { "click": ".sibbo-panel__main div[data-view=\"purposesLegInt\"] a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": "#purposesLegIntSaveAndExit" }
```

Comment:
`[data-save-and-exit]` might be more reliable (watch out, it returns multiple elements, so you'll need to make it more specific)

Reply:
Selector replaced by:
- For 1-page banner L23 -> `"click": "[data-view=\"purposes\"] .sibbo-panel__main__footer__actions [data-save-and-exit]"`
- For 2-page banner L46 -> `"click": ".sibbo-panel__main__footer__actions [data-save-and-exit]"`